com.unity.render-pipelines.high-definition/Editor/RenderPipeline/Settings/HDRenderPipelineGlobalSettingsUI.Drawers.cs

```diff
@@ -50,152 +20,6 @@ public class DocumentationUrls
             public static readonly string k_CustomPostProcesses = "Custom-Post-Process";
         }
 
-        static HDGlobalSettingsPanelIMGUI()
-        {
-            Inspector = CED.Group(
-                VolumeSection,
-                CED.Group((serialized, owner) => EditorGUILayout.Space()),
-                DiffusionProfileSettingsSection,
-                CED.Group((serialized, owner) => EditorGUILayout.Space()),
-                FrameSettingsSection,
-                CED.Group((serialized, owner) => EditorGUILayout.Space()),
-                LayerNamesSection,
-                CED.Group((serialized, owner) => EditorGUILayout.Space()),
-                CustomPostProcessesSection,
-                CED.Group((serialized, owner) => EditorGUILayout.Space()),
-                MiscSection,
-                CED.Group((serialized, owner) => EditorGUILayout.Space()),
-                ResourcesSection
-            );
-        }
-
-        SerializedHDRenderPipelineGlobalSettings serializedSettings;
-        HDRenderPipelineGlobalSettings settingsSerialized;
-
-        public void OnTitleBarGUI()
-        {
-            if (GUILayout.Button(CoreEditorStyles.iconHelp, CoreEditorStyles.iconHelpStyle))
-                Help.BrowseURL(Documentation.GetPageLink("Default-Settings-Window"));
-        }
-
-        internal static bool needRefreshVfxErrors = false;
-
-        public void DoGUI(string searchContext)
-        {
-            // When the asset being serialized has been deleted before its reconstruction
-            if (serializedSettings != null && serializedSettings.serializedObject.targetObject == null)
-            {
-                serializedSettings = null;
-                settingsSerialized = null;
-            }
-
-            if (serializedSettings == null || settingsSerialized != HDRenderPipelineGlobalSettings.instance)
-            {
-                if (HDRenderPipelineGlobalSettings.instance != null)
-                {
-                    settingsSerialized = HDRenderPipelineGlobalSettings.Ensure();
-                    var serializedObject = new SerializedObject(settingsSerialized);
-                    serializedSettings = new SerializedHDRenderPipelineGlobalSettings(serializedObject);
-                }
-                else
-                {
-                    serializedSettings = null;
-                    settingsSerialized = null;
-                }
-            }
-            else if (settingsSerialized != null && serializedSettings != null)
-            {
-                serializedSettings.serializedObject.Update();
-            }
```

Comment:
From what I can tell, this code (and the same part on URP) no longer exists. I don't really understand what this is about, but wanted to make sure if it was safe to delete.

Reply:
Basically, before we were creating a SerializedObject and a SerializedXXXRenderPipelineGlobalSettings. Now as we are using an Editor, the code from the editor will create a SerializedObject for us, and the SerializedXXXRenderPipelineGlobalSettings will be internally created by the CustomEditor, so we are good :) and a lot of code is not need

Reply:
Awesome! Thanks for the explanation :) good to see this removed.